server/pkg/client/vaultcred.go

```diff
@@ -2,47 +2,64 @@ package client
 
 import (
 	"context"
+	"fmt"
 
-	"github.com/intelops/go-common/credentials"
-	"github.com/pkg/errors"
+	"github.com/kube-tarian/kad/server/pkg/config"
+	"github.com/kube-tarian/kad/server/pkg/types"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	vaultpb "github.com/intelops/vault-cred/proto/pb/vaultcredpb"
 )
```

Comment:
vault-cred only creates vault policies and roles for services.
vault-cred GRPC APIs are added for administration purpose.

services like agent, server, iam etc.. need to use go-common/credentials package to read/write credentials to vault directly.

Reply:
This will done with another PR